src/pymatgen/util/testing/__init__.py

```diff
@@ -41,6 +42,17 @@ class PymatgenTest(TestCase):
     # dict of lazily-loaded test structures (initialized to None)
     TEST_STRUCTURES: ClassVar[dict[str | Path, Structure | None]] = dict.fromkeys(STRUCTURES_DIR.glob("*"))
 
+    @classmethod
+    def setUpClass(cls):
```

Comment:
Need to re-think this, as it would emit a warning no matter unittest-specific functions are used or now, which would be very annoying